lib/hashie/dash.rb

```diff
@@ -44,7 +44,7 @@ def self.property(property_name, options = {})
 
       unless instance_methods.map(&:to_s).include?("#{property_name}=")
         define_method(property_name) { |&block| self.[](property_name, &block) }
-        property_assignment = property_name.to_s.concat('=').to_sym
+        property_assignment = property_name.to_s.+('=').to_sym
```

Comment:
It's an interesting syntax, maybe we can just do `"#{property_name}=" instead? Easier on the eyes?